app/views/reviews/new_review.html.erb

```diff
@@ -0,0 +1,17 @@
+<h1>Create a new Review</h1>
+
+<%= form_for @review, url: {action: "create"} do |f| %>
```

Comment:
I'll need to double check if we can use form_for for this project otherwise i'll need to change to form_tag

Reply:
I'm not sure what form_for is. you'll have to give me the run down

Reply:
You'll dig them. They make things quicker.

Reply:
Not so much for me but still